src/ui/image/Image.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect, useRef } from "react"
+
+import { Loader, Button } from "ui"
+
+import csx from "./Image.scss"
+
+namespace Image {
+    export interface Props extends React.ImgHTMLAttributes<HTMLImageElement> {
+        className?: string;
+        lowQuality: string;
```

Comment:
lowQuality feature is not needed.

Reply:
It's one of load images strategy, but I removed it.